src/Evolution/Initialization/DgDomain.hpp

```diff
@@ -99,6 +99,7 @@ struct Domain {
     using simple_tags = db::AddSimpleTags<
         ::domain::Tags::Mesh<Dim>, ::domain::Tags::Element<Dim>,
         ::domain::Tags::ElementMap<Dim, Frame::Grid>,
+        ::domain::Tags::ElementMap<Dim, Frame::Inertial>,
```

Comment:
I'm confused by this commit. If you need this to compute characteristics, then how does the moving mesh code already in develop work correctly?

Reply:
Discussion on call 5/18: we don't see why this is needed

Reply:
oops, sorry this crept in from the development branch where I had included it for the scalar wave system (which needs this commit).